docker/Dockerfile

```diff
@@ -113,18 +114,6 @@ RUN echo "## Update and install packages" \
     && rm -rf /usr/share/man/??_* \
     && echo "## Done"
 
-# ------------------------------------------------------------------------------
-# Internal test suite of tools that this image provides
-COPY test_dockerimage.sh /emsdk/
-
-RUN echo "## Internal Testing of image" \
-    && /emsdk/test_dockerimage.sh \
-    && echo "## Done"
```

Comment:
I'm curious, do most docker images contains self-test code like this withing the Dockerfile?

Reply:
As far as I know, most Docker images test the image during `docker run` instead of `docker build`. Perhaps this has been added to ensure that all cache directories of Emscripten are pre-populated (since it calls `em++ --version`)?